src/main/archetype/ui.content/src/main/content/jcr_root/conf/__appId__/settings/cloudconfigs/fdm/ms-dynamics-odata-cloud-service/.content.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0"
+    jcr:primaryType="cq:Page">
+    <jcr:content
+        cq:lastModified="{Date}2018-01-29T07:02:40.895Z"
+        cq:lastModifiedBy="admin"
+        jcr:lastModified="{Date}2018-01-29T07:02:40.882Z"
+        jcr:lastModifiedBy="admin"
+        jcr:primaryType="cq:PageContent"
+        jcr:title="MS Dynamics OData Cloud Service"
```

Comment:
Can we change the title to aligned with official naming convention "Microsoft Dynamics OData Cloud Service"? All the display name shown UI should follow this format. node name still can contain ms-*.

Reply:
I think we need to make this change.

Reply:
Changed to Microsoft Dynamics.